### PR TITLE
ZER-11 Add migration createCalculators

### PR DESCRIPTION
**Pull request by PavloMS**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: db/migrate/20210409131859_create_calculators.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateCalculators < ActiveRecord::Migration[6.1]
+  def change
+    create_table :calculators do |t|
+     t.uuid :uuid, null: false
+      t.string :name
+      
+      t.timestamps
```
Comment (maksymst) [inline db/migrate/20210409131859_create_calculators.rb:7]: Please fix indentation

**Review by maksymst — CHANGES_REQUESTED** [review]
Also, change your commit message and PR title to conform a format "TICKET_ID COMMIT_MESSAGE"

File: db/migrate/20210409131859_create_calculators.rb (hunk 1 above)
Comment (PavloMS) [reply db/migrate/20210409131859_create_calculators.rb]: Made requested changes

Comment (PavloMS) [reply db/migrate/20210409131859_create_calculators.rb]: Change commit to correct format

**Comment by PavloMS** [comment]
> Also, change your commit message and PR title to conform a format "TICKET_ID COMMIT_MESSAGE"

Change commit to correct format

**Review by maksymst — APPROVED** [review]

**Review by maksymst — APPROVED** [review]

**Pull request merged**